src/app/public/modules/phone-field/types/country-data.ts

```diff
@@ -0,0 +1,6 @@
+export interface SkyCountryData {
+  name: string;
+  iso2: string;
+  dialCode: string;
+  placeholder?: string;
```

Comment:
What is meant by "placeholder" as it applies to country data?

Reply:
That is the placeholder text that appears in the input field when the country is selected. We only grab this data the first time that a country is selected and that is why it is optional.

Reply:
Gotcha. It initially stuck out to me because the word "placeholder" doesn't mean much in the context of "country data", and it focuses its existence to a specific implementation (for example, it would be odd to use the `SkyCountryData.placeholder` value in a context other than a form field's `placeholder` attribute).

I noticed that the value comes from a `intl-tel-input` function named `getExampleNumber`. What about renaming this property to `exampleNumber`? That way, it doesn't leak its implementation, and better serves the interface of "country data".

Reply:
Sounds good to me. Done